src/containers/Storage/StorageGroups/columns/columns.tsx

```diff
@@ -144,12 +144,20 @@ const diskSpaceUsageColumn: StorageGroupsColumn = {
 const groupIdColumn: StorageGroupsColumn = {
     name: STORAGE_GROUPS_COLUMNS_IDS.GroupId,
     header: STORAGE_GROUPS_COLUMNS_TITLES.GroupId,
-    width: 130,
+    width: 140,
     render: ({row}) => {
         return row.GroupId ? (
-            <InternalLink className={b('group-id')} to={getStorageGroupPath(row.GroupId)}>
-                {row.GroupId}
-            </InternalLink>
+            <div className={b('group-id-wrapper')}>
+                <InternalLink className={b('group-id')} to={getStorageGroupPath(row.GroupId)}>
+                    {row.GroupId}
+                </InternalLink>
+                <ClipboardButton
```

Comment:
Clipboard button is hidden if column is narrow enough.
Also, ellipsis for groupId is missed.